TaskModules/powershell/TaskModuleSqlUtility/SqlQueryOnTargetMachines.ps1

```diff
@@ -48,25 +48,49 @@ function Execute-SqlQueryDeployment
         # Import SQLPS Module
         Import-SqlPs
 
-        $scriptArgument = "Invoke-Sqlcmd -ServerInstance `"$serverName`" -Database `"$databaseName`" -InputFile `"$sqlFile`" "
-        $scriptArgument += $additionalArguments
+        $spaltArguments = @{
+            ServerInstance=$serverName
+            Database=$databaseName
+            InputFile=$sqlFile
+        }
+
+        # Process Additional arguments 
+        if($additionalArguments) 
+        {
+            $additionArgsArray = $additionalArguments.Split()
```

Comment:
you may need to trim each arg and value. Please reffer this  code which does the command line argument parsing same as you are doing here - https://github.com/Microsoft/vsts-agent/blob/f7e9a7ee42d0c30f0d31d449bb51d95bd6e9e0fb/src/Microsoft.VisualStudio.Services.Agent/CommandLineParser.cs